service/host.go

```diff
@@ -20,11 +20,20 @@ var (
 		evergreen.HostRunning,
 		evergreen.HostQuarantined,
 		evergreen.HostDecommissioned,
+		evergreen.HostTerminated,
 	}
 )
 
 const (
-	IncludeSpawnedHosts = "includeSpawnedHosts"
+	IncludeSpawnedHosts            = "includeSpawnedHosts"
```

Comment:
Since these are returned from host_helpers.go, I think it makes more sense to define them in the same file